### ISSUE TITLE
Unable to set environment

### ISSUE THREAD

**Issue by savy-91** [statsig-io/dart-sdk#3]
According to the [documentation](https://docs.statsig.com/guides/using-environments):

> All of our SDKs allow you to set the environment tier your app is currently in during initialization.

and

>  All SDKs accept an SDK key and an (optional) StatsigOptions dictionary of parameters. One of those options is the environment parameter, which has a tier field. The tier can be one of: "production"/"development"/"staging".

However, the `StatsigOptions` object of this SDK only has this info:

```
class StatsigOptions {
  /// The base url to use for all SDK network requests. Defaults to https://statsigapi.net/v1/
  String? api;

  /// How long (in seconds) the Statsig client waits for the initial network request. Defaults to 3 seconds
  int initTimeout;

  StatsigOptions([this.api, this.initTimeout = 3]);
}
```

**Comment by tore-statsig** [comment]
Yeah looks like this is missing.  Thanks for the PR!  Ill let the primary author @daniel-statsig sign off

**Comment by savy-91** [comment]
To people reading this issue in the future: it's now possible to set the environment but that will not be propagated to events being sent to statsig

**Comment by dan-loomb** [comment]
> To people reading this issue in the future: it's now possible to set the environment but that will not be propagated to events being sent to statsig

This has been fixed in 0.3.2. The environment from StatsigOptions will included in events being sent to Statsig
